src/tools/peptide-search/components/__tests__/PeptideSearchForm.spec.tsx

```diff
@@ -0,0 +1,95 @@
+import { createMemoryHistory } from 'history';
+import axios from 'axios';
+import MockAdapter from 'axios-mock-adapter';
+import { screen, fireEvent } from '@testing-library/react';
+
+import customRender from '../../../../shared/__test-helpers__/customRender';
+
+import PeptideSearchForm from '../PeptideSearchForm';
+
+import initialState from '../../../../app/state/rootInitialState';
+
+import { mockSuggesterApi } from '../../../../query-builder/components/__tests__/__mocks__/autocompleteWrapperData';
+
+const mock = new MockAdapter(axios);
+mock.onGet().reply(200, mockSuggesterApi.response);
+
+const aaSequence = 'ABCDEFGHIJKLMNOPQRST';
+const ntSequence = 'ATCGAGCGATAGCGAGGGAC';
+
```

Comment:
Should we try with an invalid sequence too?

Reply:
All those tests are skipped now, pending discussion point #2